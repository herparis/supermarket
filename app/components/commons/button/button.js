import styled from 'styled-components'

import { COLORS } from '../../../constants'

const Button = styled.button`
  display: flex;
  justify-content: center;
  background-color: ${COLORS.WHITE};
  color: ${COLORS.BLACK};
  padding: 10px 0 10px 0;
  font-size: 1rem;
  border-radius: 6px;
  font-family: sans-serif;
  cursor: pointer;
  
  &:hover {
    background-color: ${COLORS.PALEVIOLETRED};
  }
`

export default Button