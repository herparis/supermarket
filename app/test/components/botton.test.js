import React from 'react'
import { shallow } from 'enzyme'

import { Button } from '../../src/components/commons/button'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('Button components rendeding', () => {
  const button = shallow(<Button />)
  it('Button renders', () => {
    checkRender(button)
  })
})