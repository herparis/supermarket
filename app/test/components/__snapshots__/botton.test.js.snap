// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button components rendeding Button renders 1`] = `ShallowWrapper {}`;
